hack/tools/prowjob-gen/generator.go

```diff
@@ -192,6 +196,42 @@ func last(list any) any {
 
 		return l2.Index(l - 1).Interface()
 	default:
-		panic(fmt.Sprintf("Cannot find last on type %s", tp))
+		panic(fmt.Sprintf("cannot find last on type %s", tp))
+	}
+}
+
+func list(v ...interface{}) []interface{} {
+	return v
+}
+
+func has(needle interface{}, haystack interface{}) bool {
+	l, err := mustHas(needle, haystack)
+	if err != nil {
+		panic(err)
+	}
+
+	return l
+}
+
+func mustHas(needle interface{}, haystack interface{}) (bool, error) {
+	if haystack == nil {
+		return false, nil
+	}
+	tp := reflect.TypeOf(haystack).Kind()
+	switch tp {
+	case reflect.Slice, reflect.Array:
+		l2 := reflect.ValueOf(haystack)
+		var item interface{}
+		l := l2.Len()
+		for i := 0; i < l; i++ {
+			item = l2.Index(i).Interface()
+			if reflect.DeepEqual(needle, item) {
+				return true, nil
+			}
+		}
+
+		return false, nil
+	default:
+		return false, fmt.Errorf("cannot find has on type %s", tp)
```

Comment:
nit: the error sounds a bit strange

Reply:
it is a copy paste from sprig, not sure we should/want to start forking...

Reply:
I'm fine to keep it as is. But the error message doesn't seem to make any sense :D

Reply:
`cannot evaluate has on type %s (only arrays and slices are supported)` would make a lot more sense